Changelog.md

```diff
@@ -19,6 +19,11 @@
 * Fix array: missing pointer touch could potentially cause use of freed memory.
 * Fix array: unnecessary additional allocation due to a bug
 
+### Notable Internal API Changes
+
+* `tapAsyncK` has been moved from `Streamly.Internal.Data.Stream.Parallel` and
+  renamed, it was called `tapAsync` in the original module.
```

Comment:
`tapAsync` from `Streamly.Internal.Data.Stream.Parallel` has been moved to `Streamly.Internal.Data.Stream.IsStream` and renamed to `tapAsyncK`.